src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1

```diff
@@ -38,3 +38,30 @@ function Test-SimpleNewVm
         Clean-ResourceGroup $vmname
     }
 }
+
+<#
+.SYNOPSIS
+Test Simple Paremeter Set for New Vm with availability sets
+#>
+function Test-SimpleNewVmAvailabilitySet
+{
+    # Setup
+    $vmname = Get-ResourceName
+
+    try
+    {
+		$username = "admin01"
+		$password = "werWER345#%^" | ConvertTo-SecureString -AsPlainText -Force
+		$cred = new-object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+
+        # Common
+		$x = New-AzureRmVM -Name $vmname -Credential $cred -AvailabilitySetName $vmname
```

Comment:
So, I think we decided to use the availability set if it existed, but not to create one if it did not exist

Reply:
I'm going to prepare a fix for this. This code was written before the decision is made.